Jenkinsfile

```diff
@@ -5,10 +5,13 @@ pipeline {
         stage('Cleanup') {
             steps {
                 scmSkip(deleteBuild: true, skipPattern:'.*\\[CI-SKIP\\].*')
+                sh 'git config --global gc.auto 0'
                 sh 'rm -rf ./target'
-                sh 'rm -rf ./Paper/Paper-API ./Paper/Paper-Server ./Paper/work/Spigot/Spigot-API ./Paper/work/Spigot/Spigot-Server'
+                sh 'rm -rf ./Paper/Paper-API ./Paper/Paper-Server ./Paper/work'
```

Comment:
Um you don't want to delete this whole folder I would think.

Reply:
It’s Ok to delete it , because it’s regenerated every build, i know that because i’m always doing that before every build, You Can test it, if You want to, by removing this folder before building

Reply:
Ah ok. Got it.